code/modules/mob/living/simple_animal/vore/badboi.dm

```diff
@@ -0,0 +1,66 @@
+/mob/living/simple_animal/badboi
```

Comment:
Should be a /hostile subtype, then remove the hostile = 1, retaliate = 1 etc. Look at the hostile subtype for vars you can remove (it's at the bottom of simple_animal.dm)